JPS_VIRTUALSENSOR/AermodAgent/src/main/java/com/cmclinnovations/aermod/Aermod.java

```diff
@@ -526,8 +528,13 @@ int runAermet(Polygon scope) {
         }
     }
 
-    int createPointsFile(List<PointSource> pointSources, int simulationSrid) {
+    int createPointsFile(List<PointSource> pointSources, int simulationSrid, String pollutantId) {
         StringBuilder sb = new StringBuilder();
+
+        double maxFlowRate = 0.0;
+        double eps = 1.0e-20;
```

Comment:
Is there a reason for the choice of this number? And have you checked that it is actually using 1e-20? In my debugger it just shows up as zero